### PR TITLE
community: migrate `bigdl-llm` to `ipex-llm`

### PR DESCRIPTION
**Pull request by shane-huang**
- **Description**:  `bigdl-llm` library has been renamed to [`ipex-llm`](https://github.com/intel-analytics/ipex-llm). This PR migrates the `bigdl-llm` integration to `ipex-llm` .
- **Issue**: N/A. The original PR of `bigdl-llm` is https://github.com/langchain-ai/langchain/pull/17953 
- **Dependencies**: `ipex-llm` library
- **Contribution maintainer**: @shane-huang

Updated doc:   docs/docs/integrations/llms/ipex_llm.ipynb
Updated test:  libs/community/tests/integration_tests/llms/test_ipex_llm.py

### CODE REVIEW

File: libs/community/langchain_community/llms/ipex_llm.py [hunk 1]
```diff
@@ -10,22 +10,22 @@
 logger = logging.getLogger(__name__)
 
 
-class BigdlLLM(LLM):
```
Comment (baskaryan) [inline libs/community/langchain_community/llms/ipex_llm.py:13]: for backwards compatibility can we keep around a bigdl.py file and does 
```python
from langchain_community.llms.ipex_llm import IPEXLLM

BigdlLLM = IPEXLLM
```

that way existing code that uses this integration won't break

File: libs/community/langchain_community/llms/ipex_llm.py [hunk 2]
```diff
@@ -10,22 +10,22 @@
...
-    """Wrapper around the BigDL-LLM Transformer-INT4 model
+class IPEXLLM(LLM):
+    """Wrapper around the IPEX-LLM model
 
     Example:
         .. code-block:: python
 
-            from langchain.llms import TransformersLLM
-            llm = TransformersLLM.from_model_id(model_id="THUDM/chatglm-6b")
+            from langchain.llms import IPEXLLM
```
Comment (baskaryan) [inline libs/community/langchain_community/llms/ipex_llm.py:19]: from langchain_community.llms import IPEXLLM

Comment (baskaryan) [reply libs/community/langchain_community/llms/ipex_llm.py:19]: also you'll need to update `langchain_community/llms/__init__.py` for this to work

**Review by baskaryan — COMMENTED** [review]
can we add a redirect to `docs/vercel.json` to point from old url to new url

File: libs/community/langchain_community/llms/ipex_llm.py (hunk 1 above)
Comment (shane-huang) [reply libs/community/langchain_community/llms/ipex_llm.py:13]: Thanks for the comment. I could not use `BigdlLLM = IPEXLLM` as the imports are different. Instead I inherite from IpexLLM and override necessary methods.

**Comment by shane-huang** [comment]
> can we add a redirect to `docs/vercel.json` to point from old url to new url

added the url redirection

**Comment by shane-huang** [comment]
@baskaryan Please kindly let me know if there's anything else I need to change.

**Comment by shane-huang** [comment]
@baskaryan Could you kindly advise why Vercel failed after removing the redundant url redirect and how can I fix it?

**Pull request merged**